### ISSUE TITLE
support multiple recipients for keybase encrypt

### ISSUE THREAD

**Issue by elopez** [keybase/keybase-issues#1405]
It should be possible to use `keybase encrypt` to encrypt something for multiple recipients, see eg.

http://laurent.bachelier.name/2013/03/gpg-encryption-to-multiple-recipients/

Currently, `keybase encrypt` just uses the first recipient on the "them" parameter and ignores the rest.

**Comment by longdarc2** [comment]
This was one of the first things I tried to do on keybase and was surprised when I couldn't. For users new to encryption this greatly changes the accessibility of encryption.

**Comment by AsherMaximum** [comment]
Yeah, this kinda kills encryption functionality right now. Very surprised it wasn't there.

**Comment by chrisnash** [comment]
Was just coming to log this issue, and found this already-existing one, so am adding a comment to bump. It is a significant issue - without it, I can't include myself in the recipients of an email so I can read it when it's in my Sent folder :) Current workaround is to use gpg directly with multiple recipients - seems like it would be an easy fix since that's an easy command line to pass. I might have a go at adding it myself if I'm feeling particularly energetic.

**Comment by zQueal** [comment]
Hopefully once the node client is out this functionality will be added. Something I'd love to see is groups. So you can add those that track you, or that you're tracking into a "group" and encrypt to that "group" which would add everyone from that group to the list of those who would be encrypted to.
